libs/ui/src/components/navigator/components/header-close-button/header-close-button.tsx

```diff
@@ -24,11 +25,19 @@ const goBackRoutes = [
   ScreensEnum.EditAccount
 ];
 
-export const HeaderCloseButton: FC = () => {
+interface Props {
+  onCloseButtonPress?: OnEventFn<void>;
+}
+
+export const HeaderCloseButton: FC<Props> = ({ onCloseButtonPress }) => {
   const { navigate, goBack } = useNavigation();
   const { name } = useRoute<RouteProp<ScreensParamList>>();
 
   const onClosePress = () => {
+    if (isDefined(onCloseButtonPress)) {
```

Comment:
you could set default value for  `onCloseButtonPress = emptyFn` and then omit this checks

Reply:
But I need to do check, and I only need `onCloseButtonPress` to occur and I have `return` here